### PR TITLE
Add API call for next/previous

### PR DESCRIPTION
**Pull request by antonmolodykh**
Refs https://github.com/Screenly/screenly-ose/issues/591

### CODE REVIEW

File: viewer.py [hunk 1]
```diff
@@ -59,13 +61,20 @@ def sigusr2(signum, frame):
     load_settings()
 
 
+def sigurg(signum, frame):
+    scheduler.reverse = True
+    logging.info('USR1 received, skipping.')
```
Comment (vpetersson) [inline viewer.py]: s/USR1/USRG/g

File: server.py [hunk 2]
```diff
@@ -515,12 +515,40 @@ def post(self):
...
+        ],
+        'responses': {
+            '200': {
+                'description': 'Asset switched'
+            }
+        }
+    })
+    def get(self, command):
+        if command == "next":
+            sh.sudo('systemctl', 'kill', '--signal=SIGUSR1', '--kill-who=main', 'screenly-viewer.service')
```
Comment (vpetersson) [inline server.py]: No sudo needed.

File: server.py [hunk 3]
```diff
@@ -515,12 +515,40 @@ def post(self):
...
+            return "Asset switched"
+        if command == "previous":
+            sh.sudo('systemctl', 'kill', '--signal=SIGURG', '--kill-who=main', 'screenly-viewer.service')
```
Comment (vpetersson) [inline server.py]: same.

**Comment by vpetersson** [comment]
Let's restructure this a bit:

 * Let's change SIGUSR2 (reload) to SIGHUP instead, which is more industry standards
 * Let's use SIGUSR2 for going backward.

File: server.py (hunk 2 above)
Comment (vpetersson) [reply server.py]: Also, let's use avoid using the `sh` module for system calls and instead use the [OS library] (https://docs.python.org/3/library/os.html)

Comment (antonmolodykh) [reply server.py]: hmm. it seems that for systemctl calls we still need sudo

Comment (vpetersson) [reply server.py]: Why do we need to use `systemctl`?  It's just a regular kill signal. For instance, you could do `pkill -SIGUSR1 -f viewer.py`, but ideally you'd do it in [native python](https://stackoverflow.com/questions/2940858/kill-process-by-name).

**Pull request merged**